eng/pipelines/common/global-build-job.yml

```diff
@@ -0,0 +1,47 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osSubgroup: ''
+  container: ''
+  testGroup: ''
+  crossrootfsDir: ''
+  timeoutInMinutes: ''
```

Comment:
Trim unused variables (to avoid a false assumption that they work when someone wants to add more legs later)?

Reply:
Fixed.